Centered for #141

jordansread opened the pull request:
attempt at #141, but didn't try to do too much

Commit:
Centered for #141

Commit:
code for making fig 2 prettier

Commit:
#154

Commit:
for #155

jread-usgs commented:
@eread-usgs review? Note that I think this change makes fig 3 too small relative to 2. Fixing that I think will need some Marty help

jread-usgs commented:
oh joy

jread-usgs commented:
will fix conflicts

Commit:
merge conflict

Merge branch 'master' of github.com:USGS-VIZLAB/great-lakes-microplastics

# Conflicts:
#	layout/css/main.css

jread-usgs commented:
@ldecicco-USGS review?
